modules/benchmark/R/load.data.R

```diff
@@ -16,6 +16,7 @@ load.data <- function(data.path, format, start_year = NA, end_year=NA, site=NA,
   require(PEcAn.benchmark)
   require(lubridate)
   require(udunits2)
+  require(dplyr)
```

Comment:
FWIW, the official line seems to be that packages generally shouldn't use `require` or `library` at all (see [here](http://r-pkgs.had.co.nz/namespace.html#search-path) and [here](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-Dependencies)). Instead, just make sure the required package is listed in `DESCRIPTION` under `Imports:`, and access external functions explicitly using `package::function`. (An alternative to the verbose `::` syntax is to import the external function into the namespace via the `NAMESPACE` file, which can be handled by Roxygen if you use `@importFrom` or `@import`.)